src/landscape.cpp

```diff
@@ -1453,11 +1431,7 @@ static std::tuple<bool, bool> FlowRiver(TileIndex spring, TileIndex begin, uint
 				/* We only want a lake if the river is long enough. */
 				DistanceManhattan(spring, lakeCenter) > min_river_length) {
 			end = lakeCenter;
```

Comment:
Unrelated to the PR, but that's a "nice" abuse of a local var to backup another local var.

Reply:
ah, so it can be removed?

Reply:
no it's still needed

Reply:
nevermind, it can't, it's used below